### PR TITLE
feat: add coingecko token list to Gnosis chain

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

The list: https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/GnosisCoingeckoTokensList.json

Also updated the token list logo: https://github.com/cowprotocol/token-lists/pull/385

image

# To Test

The new list should be enabled in Gnosis chain by default

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @shoom3301 , should this list be available for the widget configurator as well?

**Comment by shoom3301** [comment]
@elena-zh yes it should, since widget it's actually Cowswap :)

File: libs/tokens/src/updaters/TokensListsUpdater/index.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { TOKENS_LISTS_UPDATER_INTERVAL, getFulfilledResults, getIsTimeToUpdate }
 
 const { atom: lastUpdateTimeAtom, updateAtom: updateLastUpdateTimeAtom } = atomWithPartialUpdate(
   atomWithStorage<Record<SupportedChainId, number>>(
-    'tokens:lastUpdateTimeAtom:v0',
+    'tokens:lastUpdateTimeAtom:v1',
```
Comment (anxolin) [inline libs/tokens/src/updaters/TokensListsUpdater/index.ts:20]: Why does this need to be updated?

Comment (shoom3301) [reply libs/tokens/src/updaters/TokensListsUpdater/index.ts:20]: We update token lists sources only once in 6 hours.
If we don't change the atom version, the new list will appear only in 6 hours

Comment (shoom3301) [reply libs/tokens/src/updaters/TokensListsUpdater/index.ts:20]: image

Comment (anxolin) [reply libs/tokens/src/updaters/TokensListsUpdater/index.ts:20]: thanks for the response

**Pull request merged**